integration-intygsadmin/src/test/java/se/inera/intyg/minaintyg/integration/intygsadmin/BannerFilterServiceTest.java

```diff
@@ -0,0 +1,91 @@
+package se.inera.intyg.minaintyg.integration.intygsadmin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.minaintyg.integration.api.banner.model.Application;
+import se.inera.intyg.minaintyg.integration.intygsadmin.client.dto.BannerDTO;
+
+@ExtendWith(MockitoExtension.class)
+class BannerFilterServiceTest {
+
+  @InjectMocks
+  private BannerFilterService bannerFilterService;
+
+  @Test
+  void shouldFilterOnNullValuesForApplication() {
+    final var bannerDTOS = new BannerDTO[]{
```

Comment:
Maybe some extra lines between preparation, action and check in the test will increase the readability